.github/workflows/deploy.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     - name: Prepare
       id: prep
       run: |
-        IMG=quay.io/joelanford/helm-operator
+        IMG=quay.io/operator-framework/helm-operator-plugins
```

Comment:
TODO: I think we probably should just skip the deploy step for now since the actual image is being released from the main SDK repo, I didn't want to rename this to `quay.io/operator-framework/helm-operator` out of fear we may accidentally overwrite the real image before we were ready